gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/matching/common/PostProcessor.java

```diff
@@ -66,11 +67,12 @@ public static GraphCollection extractGraphCollection(
    */
   public static GraphCollection extractGraphCollection(
     DataSet<EPGMElement> elements, GradoopFlinkConfig config, boolean mayOverlap) {
+    GraphCollectionFactory factory = config.getGraphCollectionFactory();
```

Comment:
We change the signature of this method here. Instead of the config, we are using a ElementFactoryProvider.